google-cloud-spanner-executor/pom.xml

```diff
@@ -93,7 +93,10 @@
     <dependency>
       <groupId>com.google.api.grpc</groupId>
       <artifactId>proto-google-cloud-spanner-executor-v1</artifactId>
```

Comment:
Shouldn't we also remove `proto-google-cloud-spanner-executor-v1` dependency? I am assuming this the proto repository which we no longer need.

Reply:
Earlier we had this `artifactId` getting uploaded from [cndb-protos](https://github.com/googleapis/cndb-client-testing-protos/blob/db032a52db722a87522733cff3e314eed02ab600/pom.xml#L6) repo.
Now we will do it through the `proto-google-cloud-spanner-executor-v1 ` which is in [java-spanner](https://github.com/googleapis/java-spanner/tree/main/proto-google-cloud-spanner-executor-v1) itself.